docs/development.md

```diff
@@ -35,3 +35,39 @@ Since this is very unlikely and most times an abs path is not strictly necessary
 "Should" functions can be used anywhere,
 but care should be taken not to overuse them when a fallback value is not sufficient.
 "Must" variants should be favored in testing code, since they will more obviously fail tests.
+
+## Cross-Platform Scripting
+
+We are writing bash scripts with the intention of running them in a POSIX environment.
+However, differences between operating systems, particularly between Linux and MacOS,
```

Comment:
Instead of Linux and MacOS, a more correct pairing would be Linux and BSD (which MacOS) is based on.
Note that this applies to the whole PR.